tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

```diff
@@ -107,6 +107,14 @@ export class LineChartComponent implements AfterViewInit, OnChanges, OnDestroy {
   @Input()
   tooltipTemplate?: TooltipTemplate;
 
+  /**
+   * Optional parameter to tweak whether to propagate update to line chart implementation.
+   * When not specified, it defaults to `true`. When it is `false`, it remembers what has
+   * changed and applies the change when the update is enabled.
+   */
+  @Input()
+  enableUpdate?: boolean;
```

Comment:
Could we invert the meaning and have a property `disableUpdates` instead?  My default assumption when reading optional properties is that `enableXYZ = false` would be equivalent to not specifying a value.

Then L319 can be `if (!this.lineChart || this.disableUpdate) return;`

Reply:
I generally prefer positive APIs but I can see a case of `disableUpdate` here, too.

I do not exactly recall but I think there was ToTT or code style guide around this. It says something like how `disableUpdate=false` making you think more.